app/controllers/interaction_claims_controller.rb

```diff
@@ -55,4 +51,40 @@ def perform_interaction_search
     search_results = LookupInteractions.find(params)
     @search_results = InteractionSearchResultsPresenter.new(search_results, view_context)
   end
+
+  def unpack_locals params
```

Comment:
Probably want to add `()` around params here. Its syntactically valid, but by convention, parens are always used in method declarations, and optional when calling methods.